homeassistant/components/sensor/starlingbank.py

```diff
@@ -0,0 +1,103 @@
+"""
+Support for balance data via the Starling Bank API.
+
+For more details about this platform, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.starlingbank/
+"""
+import logging
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['starlingbank==1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+BALANCE_TYPES = ['cleared_balance', 'effective_balance']
+
+CONF_ACCOUNTS = 'accounts'
+CONF_BALANCE_TYPES = 'balance_types'
+CONF_SANDBOX = 'sandbox'
+
+DEFAULT_SANDBOX = False
+DEFAULT_ACCOUNT_NAME = 'Starling'
+
+ICON = 'mdi:currency-gbp'
+
+ACCOUNT_SCHEMA = vol.Schema({
+    vol.Required(CONF_ACCESS_TOKEN): cv.string,
+    vol.Optional(CONF_BALANCE_TYPES, default=BALANCE_TYPES):
+        vol.All(cv.ensure_list, [vol.In(BALANCE_TYPES)]),
+    vol.Optional(CONF_NAME, default=DEFAULT_ACCOUNT_NAME): cv.string,
+    vol.Optional(CONF_SANDBOX, default=DEFAULT_SANDBOX): cv.boolean,
+})
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ACCOUNTS): vol.Schema([ACCOUNT_SCHEMA]),
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
```

Comment:
We have renamed `add_devices` to `add_entities`.

Reply:
Thanks @MartinHjelmare, I think I'm write in saying this is just a cosmetic thing? If so, would it be a good idea for me to wrap this rename up in a bigger change at a later date?

Reply:
Yes, nothing is broken like this. :+1:

But it's nice to keep everything in sync to allow future smooth migrations.